app/models/conversion_host.rb

```diff
@@ -84,30 +84,29 @@ def get_conversion_log(path)
   end
 
   def check_conversion_host_role
-    install_conversion_host_module
     playbook = "/usr/share/ovirt-ansible-v2v-conversion-host/playbooks/conversion_host_check.yml"
-    extra_vars = { :v2v_manageiq_conversion_host_check => true }
-    ansible_playbook(playbook, extra_vars)
+    ansible_playbook(playbook)
     tag_resource_as('enabled')
   rescue
     tag_resource_as('disabled')
   end
 
-  def enable_conversion_host_role(v2v_vddk_package_url = nil)
-    install_conversion_host_module
-    v2v_vddk_package_url ||= "http://#{resource.ext_management_system.hostname}/vddk/VMware-vix-disklib-stable.tar.gz"
+  def enable_conversion_host_role(vmware_vddk_package_url = nil, vmware_ssh_private_key = nil)
+    raise "vddk_package_url is mandatory if transformation method is vddk" if vddk_transport_supported && vmware_vddk_package_url.nil?
+    raise "ssh_private_key is mandatory if transformation_method is ssh" if ssh_transport_support && vmware_ssh_private_key.nil?
```

Comment:
`ssh_transport_support` should be `ssh_transport_supported` here, yes?

Reply:
Good catch. Fixed.